Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AdjustableHeaderWidget.h>
+#include <DownloadController.h>
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <CreateAGemScreen.h>
+#include <ProjectUtils.h>
+#include <PythonBindingsInterface.h>
+
+
+#include <PythonBindingsInterface.h>
+#include <QDir>
+#include <QFileDialog>
+#include <QHBoxLayout>
+#include <QHash>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStackedWidget>
+#include <QStandardPaths>
+#include <QVBoxLayout>
+#include <QRadioButton>
+#include <QLabel>
+#include <QDialogButtonBox>
+
+
+namespace O3DE::ProjectManager
+{
+    CreateAGemScreen::CreateAGemScreen(QWidget* parent)
+        : ScreenWidget(parent)
+    {
+        QVBoxLayout* vLayout = new QVBoxLayout();
+        vLayout->setSpacing(0);
+        vLayout->setContentsMargins(0, 0, 0, 0);
+
+        QScrollArea* firstScreenScrollArea = CreateFirstScreen();
+        QScrollArea* secondScreenScrollArea = CreateSecondScreen();
+        QScrollArea* thirdScreenScrollArea = CreateThirdScreen();
+
+        connect(m_gemDisplayName->lineEdit(), &QLineEdit::textChanged, this, &CreateAGemScreen::OnGemDisplayNameUpdated);
```

Comment:
Following the design, we only need the messages to appear once next has been clicked, we could just run validation at that point to save running it every time the text has changed, and also saves connecting lots of signals.

Reply:
These have been removed; the error states will now only disappear once there is text in the required field and the user hits the next button.